src/components/NcAppNavigation/NcAppNavigation.vue

```diff
@@ -183,6 +189,13 @@ export default {
 		gap: var(--default-grid-baseline, 4px);
 		padding: calc(var(--default-grid-baseline, 4px) * 2);
 	}
+	&__content {
+		height: 100%;
+		display: flex;
+		flex-direction: column;
+		flex-grow: 0;
+		flex-shrink: 0;
+	}
```

Comment:
Isn't it out of scope of this PR?

Reply:
no, styles for wrapper (`app-navigation__content`) has to be fixed

Reply:
> Isn't this out of scope of this PR?

Adding a new container for content breaks the layout set by the original container

```
# before

.app-navigation   --- a flex container
|-- default slot     --- flex container's content
|-- ul               --- flex container's content
|-- footer slot      --- flex container's content

# after

.app-navigation              --- an old flex container
|-- .app-navigation__content   --- the new flex container
|-- |-- default slot             --- flex container's content
|-- |-- ul                       --- flex container's content
|-- |-- footer slot              --- flex container's content
```

So all flex container-related styles should be moved.

> it is hard to test all apps and the risk to break something is high.

@JuliaKirschenheuter It seems to me, that it cannot be broken. The content was in the flex container, it is still in the flex container with the same layout. The old container now has only one element.

The only thing that may be broken is if some app set styles to the root of the component. But it will break anyway now.

Reply:
you are right, thanks a lot!